library/src/CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ rocm_install_targets(
 rocm_export_targets(
   TARGETS roc::rocfft
   PREFIX rocfft
-  DEPENDS PACKAGE hip
+  DEPENDS PACKAGE HIP
```

Comment:
The package config file is name is [hip-config.cmake](https://github.com/ROCm-Developer-Tools/HIP/blob/master/CMakeLists.txt#L294) so it shouldn't be upper case as that would be the opposite of case sensitive.

Reply:
Right, it should stay lower case. Thank you.